IrConsoleApp/Program.cs

```diff
@@ -3,15 +3,149 @@
 
 namespace IrynaZharnasek_SaysHello
```

Comment:
Please align file lines by pressing ctrl+K and +D keys combination on the keyboard

Reply:
Done